workspace-service/Makefile

```
@@ -1,6 +1,8 @@
 app := workspace-service
 image := fnhsproduction.azurecr.io/$(app)
 tag := latest
+localdb := postgres://postgres:postgres@localhost:5432
+db := $(kubectl get secret -n workspace-service workspace-service -o jsonpath='{.data.url}' | base64 -D)
```

```suggestion
db := $$(kubectl get secret -n workspace-service workspace-service -o jsonpath='{.data.url}' | base64 -D)
```
My instinct says this but I've not checked it.

---

yeah I'd say so